Containerfile-fedora-amd64

```diff
@@ -1,15 +1,17 @@
-FROM quay.io/fedora/fedora-minimal:40 as builder
-ARG name=Fedora
```

Comment:
I noticed `name` isn't used in the steps. Does it add any metadata you want or is it a leftover from earlier code?

Reply:
Probably a leftover, I was experimenting quite a bit trying to figure out a common containerfile for both platforms but was unable to find one.